src/components/DashBoard.js

```diff
@@ -12,6 +12,8 @@ import Grid from '@material-ui/core/Grid';
 import Paper from '@material-ui/core/Paper';
 import Typography from '@material-ui/core/Typography';
 import Box from '@material-ui/core/Box';
+import Slide from '@material-ui/core/Slide';
+import styled from 'styled-components';
```

Comment:
If you make this `import styled from 'styled-components/macro`, this will inject the Styled components variable into the auto-generated CSS className